packages/css/src/link/link.scss

```diff
@@ -6,45 +6,41 @@
 @import "../../utils/breakpoint";
 
 .amsterdam-link {
-  color: var(--amsterdam-link-color, blue);
-  font-family: var(--amsterdam-paragraph-font-family);
+  color: var(--amsterdam-link-color);
+  font-family: var(--amsterdam-link-font-family);
```

Comment:
Discussed this. Conclusion: yes, components must only use their own tokens to make reuse easier. E.g. another implementer of our components may want to have different font families for paragraph and link and this would not be possible if links use the paragraph font tokens.